sample_app/app/controllers/users_controller.rb

```diff
@@ -8,6 +8,7 @@ def show
   def create
   	@user = User.new(user_params)
   	if @user.save
+      log_in @user
```

Comment:
Yes. When a new user gets created in the system we automatically log them in. I think in a later chapter we get rid of that in favor of waiting on email verification first.